frontend/src/lib/dashboard/getQueryResults.ts

```diff
@@ -87,4 +87,6 @@ export interface GetQueryResultsProps {
 		pagination?: Pagination;
 		selectColumns?: any;
 	};
+	start?: number;
+	end?: number;
```

Comment:
didn't we already have start and end?

Reply:
Yes, We have start and end time which we use to calculate using prepareQueryRangePayload which further calls getStartEndRangeTime. This depends on what you pass for selectedTime and globalSelectedInterval. But in this case, I have to pass the start and end time based on the log line timestamp.